### DISCUSSION TITLE
Support MBED OS6

### DISCUSSION THREAD

**Discussion by rainmaker64** [Azure/azure-iot-sdk-c#2180] [Q&A]
Hi,

I have been using azure_iot_sdk-c in MBED OS-5 platform well.
But I have to change the hardware platform which supports only MBED OS-6.
And I finished porting azure_iot_sdk-c into MBED OS-6 and it succeeded in connecting to azure IoT hub.
But when it receives the message from the server it fails and disconnects the session.
I am wondering which points I have to check to solve this issue.

Thanks
Seunghyun Jung

**Comment by ewertons** [comment]
Hi @rainmaker64, MBED OS support has been discontinued in the azure-iot-sdk-c.

Having said that, you could check on the azure-iot-sdk-c verbose logs what messages it prints when the error occurs, or if there are any error logs printed at all.
You can enable verbose logs in the azure-iot-sdk-c by following this example: https://github.com/Azure/azure-iot-sdk-c/blob/main/iothub_client/samples/iothub_convenience_sample/iothub_convenience_sample.c#L323

**Reply by ewertons** [reply]
We are going to close this discussion since it has been waiting for too long, but please feel free to reopen it if you would like to follow up.
